### ISSUE TITLE
cannot run test/manual-testing/AudioContext-sound-output.js

### ISSUE THREAD

**Issue by wfwalker** [sebpiq/node-web-audio-api#42]
After following the instructions in the README, I try to run this manual test and I get this:

```
$ node test/manual-testing/AudioContext-sound-output.js 

module.js:340
    throw err;
    ^
Error: Cannot find module 'speaker'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/walker/Sites/node-web-audio-api/test/manual-testing/AudioContext-sound-output.js:5:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
```

Same behavior observed on both Raspberry PI 2 running Raspbian as well as Mac OS X 10.10.2

**Comment by sebpiq** [comment]
Yes, node-speaker used to be a dependency, but now it isn't anymore, instead node-web-audio-api proviodes an out stream that you can use to write raw PCM data anywhere you want : https://github.com/sebpiq/node-web-audio-api#selecting-an-audio-output

So install speaker before trying this example.

**Comment by wfwalker** [comment]
Great, that worked for me on both devices! I see how that section of the README is right after the demo instructions I followed, I think one more sentence in there might make the whole thing clearer.

**Comment by sebpiq** [comment]
Oh right ... I see. I'll add this instruction there...

On Wed, May 20, 2015 at 5:47 PM, Bill Walker <notifications@github.com>
wrote:

> Closed #42 <https://github.com/sebpiq/node-web-audio-api/issues/42>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sebpiq/node-web-audio-api/issues/42#event-309595662>.
>



-- 

*Sébastien Piquemal*

 -----* @sebpiq*
 ----- http://github.com/sebpiq
 ----- http://funktion.fm
